src/codebook/controls/filters/init.js

```diff
@@ -24,18 +24,17 @@ export function init(codebook) {
   });
 
   //Clear custom controls.
-  selector.selectAll("ul.nav").remove();
+  selector.selectAll("ul.filter-list").remove();
 
   //Add filter controls.
-  var filterList = selector.append("ul").attr("class", "nav");
-
+  var filterList = selector.append("ul").attr("class", "filter-list");
   var filterItem = filterList
     .selectAll("li")
     .data(codebook.config.filters)
     .enter()
     .append("li")
     .attr("class", function(d) {
-      return "custom-" + d.key + " filterCustom";
+      return "custom-" + d.value_col + " filterCustom";
```

Comment:
Does this affect how the user specifies the filters object, i.e. `{key:` vs `{value_col:`?